library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt

```diff
@@ -49,7 +50,7 @@ class DonutProgressView @JvmOverloads constructor(
      */
     var masterProgress: Float = 1f
         set(value) {
-            if (value !in (0f..1f)) {
+            if (value !in 0f..1f) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/3f8afbf9c8b622c5d946604681947fabe1098940/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L53"></span>Parentheses in (0f..1f) are unnecessary and can be replaced with: 0f..1f</td>
    </tr>
  </tbody>
</table>